app/stac_api/views.py

```diff
@@ -363,7 +364,9 @@ def get_queryset(self):
 
     def get(self, request, *args, **kwargs):
         queryset = self.filter_queryset(self.get_queryset())
-
+        if not Collection.objects.filter(name=self.kwargs['collection_name']).exists() or \
+                not Item.objects.filter(name=self.kwargs['item_name']).exists():
+            raise Http404("This collection does not exists.")
```

Comment:
Same as above. Also from code style using `()` instead of `\` make the code cleaner in my opinion.